enterprise/app/trends/trends.tsx

```diff
@@ -177,6 +178,14 @@ export default class TrendsComponent extends React.Component<Props, State> {
     return moment(date).format("MMM D");
   }
 
+  getStat(date: string): invocation.TrendStat {
+    return this.state.dateToStatMap.get(date) || new invocation.TrendStat();
+  }
```

Comment:
If you switch back to `ITrendStat` then you could write

```
  getStat(date: string): invocation.TrendStat {
    return this.state.dateToStatMap.get(date) || {};
  }
```

and I think the type checker should recognize `{}` as being assignable to `ITrendStat`, not 100% sure though. Same in the func below